modules/nextflow/src/main/groovy/nextflow/k8s/K8sTaskHandler.groovy

```diff
@@ -16,6 +16,8 @@
 
 package nextflow.k8s
 
+import java.nio.file.Paths
```

Comment:
is this import used? i dont see it in any of the changes

Reply:
no, i think i need to enable remove unused imports